src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/EvaluateConcordanceOnNonGenotypedSVVCFs.java

```diff
@@ -0,0 +1,497 @@
+                } else if (classifiedVariant instanceof FilteredTrueNegative) {
+                    other = StringUtil.join(",", classifiedVariant.getEval().getFilters());
+                    supportScore = 1.0f;
+                } else {
+//                        (classifiedVariant instanceof TruePositive || classifiedVariant instanceof FilteredFalseNegative)
+                    other = ConcordanceState.TRUE_POSITIVE.getAbbreviation(); // to pad so that all fields have same number of fields
+                    supportScore = ((BiVariant) classifiedVariant).getSupportScore();
+                }
+                final String concordance = classifiedVariant.getConcordanceState().getAbbreviation();
+                final String evalID = eval.getID();
+                final String contigNames = eval.hasAttribute(GATKSVVCFConstants.CONTIG_NAMES)
+                        ? SVUtils.getAttributeAsStringStream(eval, GATKSVVCFConstants.CONTIG_NAMES).collect(Collectors.joining(","))
+                        : NA_STRING;
+                writer.write(String.format("%s\t%d\t%d\t%s\t%.2f\n", eval.getContig(), eval.getStart(), eval.getEnd(),
+                        concordance + ";" + type + ";" + other + ";" + evalID + ";" + contigNames, supportScore));
+            }
+        } catch (final IOException ioe) {
+            throw new GATKException("Can't write BED file " + outPrefix + "concordance.bed", ioe);
+        }
+    }
+
+    private void writeSkippedEvalBedFile() {
+        try (final OutputStreamWriter writer = new OutputStreamWriter(new BufferedOutputStream(
+                BucketUtils.createFile(outPrefix + "skipped.bed")))) {
+            for (final VariantContext skipped : evalSkippedTesting) {
+                final String asmCtg;
+                if(skipped.hasAttribute(GATKSVVCFConstants.CONTIG_NAMES)) {
+                    asmCtg = StringUtil.join(",", SVUtils.getAttributeAsStringList(skipped, GATKSVVCFConstants.CONTIG_NAMES));
+                } else {
+                    asmCtg = NA_STRING;
+                }
+                StringBuilder stringBuilder = new StringBuilder(skipped.getContig());
+                stringBuilder.append("\t").append(skipped.getStart());
+                stringBuilder.append("\t").append(skipped.getEnd());
+                stringBuilder.append("\t").append(skipped.getID())
+                        .append(";").append(asmCtg)
+                        .append(";").append(skipped.getAttributeAsString(VARIANT_SKIPPED_EVAL_REASON_KEY, ""))
+                        .append("\n");
+                writer.write( stringBuilder.toString() );
+            }
+        } catch (final IOException ioe) {
+            throw new GATKException("Can't write BED file " + outPrefix + "concordance.bed", ioe);
+        }
+    }
+
+    // TODO: 6/3/18 preferably, this should be in htsjdk
+    private enum StructVarType {
```

Comment:
There is several places in the GATK code that we have such a enum... can we consolidate them now?

Reply:
now removed